src/controllers/students.controller.js

```diff
@@ -0,0 +1,32 @@
+const httpStatus = require('http-status');
+const ApiError = require('../utils/ApiError');
+const catchAsync = require('../utils/catchAsync');
+const { createStudentNotification } = require('../utils/notifications');
+const { userService, tokenService, notificationService } = require('../services');
+
+const createStudent = catchAsync(async (req, res) => {
+    const student = await userService.createUser(req.body);
```

Comment:
you should update role to 'student' like ` const student = await userService.createUser({ ...req.body, role : 'student' });`

Reply:
you also have 2 more spaces before 'const'. You disabled eslintrc.json. therefore you would not know these tiny errors.